flows/events/msg_received.go

```diff
@@ -8,38 +8,40 @@ import (
 // TypeMsgReceived is a constant for incoming messages
 const TypeMsgReceived string = "msg_received"
 
-// MsgReceivedEvent events are used for resuming flows or starting flows. They represent an MO
-// message for a contact.
+// MsgReceivedEvent events are used for starting flows or resuming flows which are waiting for a message.
+// They represent an MO message for a contact.
 //
 // ```
 //   {
-//    "type": "msg_received",
-//    "step_uuid": "8eebd020-1af5-431c-b943-aa670fc74da9",
-//    "created_on": "2006-01-02T15:04:05Z",
-//    "urn": "tel:+12065551212",
-//    "channel_uuid": "61602f3e-f603-4c70-8a8f-c477505bf4bf",
-//    "contact_uuid": "0e06f977-cbb7-475f-9d0b-a0c4aaec7f6a",
-//    "text": "hi there"
+//     "type": "msg_received",
+//     "created_on": "2006-01-02T15:04:05Z",
+//     "urn": "tel:+12065551212",
+//     "channel_uuid": "61602f3e-f603-4c70-8a8f-c477505bf4bf",
+//     "contact_uuid": "0e06f977-cbb7-475f-9d0b-a0c4aaec7f6a",
+//     "text": "hi there",
+//     "attachments": ["https://s3.amazon.com/mybucket/attachment.jpg"]
```

Comment:
In RapidPro we currently only expose the URL part (so that's all we currently send to goflow), but we could do something like what we do for URNs here:

```
@run.input.attachments[1] => "image/jpeg:https://s3.amazon.com/mybucket/attachment.jpg"
@run.input.attachments[1].url => "https://s3.amazon.com/mybucket/attachment.jpg"
@run.input.attachments[1].content_type => "image/jpeg"
```